src/features/comment/CommentEllipsis.tsx

```diff
@@ -87,6 +88,23 @@ export default function MoreActions({ comment, rootIndex }: MoreActionsProps) {
   const mySaved = commentSavedById[comment.comment.id] ?? comment.saved;
 
   const isMyComment = getRemoteHandle(comment.creator) === myHandle;
+  const commentExists = !comment.comment.deleted && !comment.comment.removed;
+  const localMod = isLocalModerator(comment.community);
+
+  let selectTextOption = undefined;
+  if (commentExists) {
+    selectTextOption = {
+      text: "Select Text",
+      role: "select-text",
+      icon: textOutline,
+    };
+  } else if (isMyComment || localMod) {
+    selectTextOption = {
+      text: "Select Deleted Text",
```

Comment:
For consistency, let's just not show "select text" if the comment isn't actually shown, and ignore moderators and "is my comment" for now. I know we'll want to revisit this with the mod work.

Reply:
Sounds good!